src/components/Dashboard/dashboard.module.scss

```
@@ -0,0 +1,77 @@
+.dashboard { 
+    border-left: #dad8d8 solid 2px;
+    border-radius: 3px;
+    padding-left: 3%;
+    margin: 1%;
+}
+.dashboardContainer {
+    overflow-y: scroll;
+    overflow-x: hidden;
+    height: 800px;
+}
+
+.dashboardNav { 
+    display: flex;
+    justify-content: space-between;
+    --width: 100%;
+    padding: 1%;
+}
+
+.dashboardReportCard { 
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 1rem;
+    border: #dad8d8 solid 1px;
+    border-radius: 3px;
+    padding: 3%;
+}
+
+.dashboardTitle { 
+    width: 50%;
+    padding-top: 20px;
```

rem preferred here too pls 
```suggestion
    padding-top: 2rem;
```